### PR TITLE
multi: Run gofmt on the entire repository

### PR DESCRIPTION
**Pull request by kcalvinalvin**
The doc formatting changes introduced in the recent go version is increasing the diff for all of the new commits.  Formatting it all in this commit will help the readability of future PRs by reducing the diff.

### CODE REVIEW

File: btcutil/coinset/coins.go [hunk 1]
```diff
@@ -75,7 +75,8 @@ func (cs *CoinSet) TotalValue() (value btcutil.Amount) {
 }
 
 // TotalValueAge returns the total value * number of confirmations
-//  of the coins in the set.
+//
+//	of the coins in the set.
```
Comment (guggero) [inline btcutil/coinset/coins.go:79]: This looks like an accidental indentation.

File: integration/csv_fork_test.go [hunk 2]
```diff
@@ -95,17 +95,22 @@ func makeTestOutput(r *rpctest.Harness, t *testing.T,
...
-//      rejected from the mempool.
-//    - Transactions within non-final MTP based lock-times should be accepted
-//      in valid blocks.
 //
-//  - Post soft-fork:
-//    - Transactions with non-final lock-times from the PoV of MTP should be
-//      rejected from the mempool and when found within otherwise valid blocks.
-//    - Transactions with final lock-times from the PoV of MTP should be
-//      accepted to the mempool and mined in future block.
+//   - Pre soft-fork:
```
Comment (guggero) [inline integration/csv_fork_test.go:99]: I think these need some manual adjustments to preserve the different levels of indentation.

File: integration/csv_fork_test.go [hunk 3]
```diff
@@ -391,13 +396,13 @@ func assertTxInBlock(r *rpctest.Harness, t *testing.T, blockHash *chainhash.Hash
...
 //
 // Overview:
-//  - Pre soft-fork:
-//    - A transaction spending a CSV output validly should be rejected from the
-//    mempool, but accepted in a valid generated block including the
-//    transaction.
-//  - Post soft-fork:
-//    - See the cases exercised within the table driven tests towards the end
-//    of this test.
+//   - Pre soft-fork:
```
Comment (guggero) [inline integration/csv_fork_test.go:399]: Same here.

File: mempool/doc.go [hunk 4]
```diff
@@ -31,40 +31,40 @@ proceed.  Typically, this will involve things such as relaying the transactions
...
-   - Max orphan transaction size
-   - Max number of orphan transactions allowed
- - Additional metadata tracking for each transaction
-   - Timestamp when the transaction was added to the pool
-   - Most recent block height when the transaction was added to the pool
-   - The fee the transaction pays
-   - The starting priority for the transaction
- - Manual control of transaction removal
-   - Recursive removal of all dependent transactions
+  - Maintain a pool of fully validated transactions
```
Comment (guggero) [inline mempool/doc.go:39]: And here.

File: peer/doc.go [hunk 5]
```diff
@@ -16,49 +16,49 @@ Payment Verification (SPV) nodes, proxies, etc.
...
- - Helper functions pushing addresses, getblocks, getheaders, and reject
-   messages
-   - These could all be sent manually via the standard message output function,
-     but the helpers provide additional nice functionality such as duplicate
-     filtering and address randomization
- - Ability to wait for shutdown/disconnect
- - Comprehensive test coverage
-
-Peer Configuration
+  - Provides a basic concurrent safe bitcoin peer for handling bitcoin
```
Comment (guggero) [inline peer/doc.go:19]: More multi-level indentations.

**Review by guggero — COMMENTED** [review]
Nice, thanks!
Do we also need to bump the Go version of the main `go.mod` so these changes aren't reverted by someone using Go 1.17?

**Comment by kcalvinalvin** [comment]
Addressed all the aforementioned formatting issues in the latest push.

**Review by Roasbeef — APPROVED** [review]
LGTM 🐲

File: blockchain/chain.go [hunk 6]
```diff
@@ -386,7 +387,7 @@ func (b *BlockChain) calcSequenceLock(node *blockNode, tx *btcutil.Tx, utxoView
 	// return sequence lock values of -1 indicating that this transaction
 	// can be included within a block at any given height or time.
 	mTx := tx.MsgTx()
-	sequenceLockActive := mTx.Version >= 2 && csvSoftforkActive
+	sequenceLockActive := uint32(mTx.Version) >= 2 && csvSoftforkActive
```
Comment (0xB10C) [inline blockchain/chain.go:390]: :sunglasses:

**Pull request merged**